react-native/src/ItemSchema.js

```diff
@@ -1,14 +1,18 @@
 import {BSON} from 'realm';
-import {createRealmContext} from '@realm/react';
 
-export class Task {
+export class Item {
   constructor({id = new BSON.ObjectId(), isComplete = false}) {
     this.isComplete = isComplete;
     this._id = id;
   }
 
   static schema = {
+    // :state-start: partition-based-sync
     name: 'Task',
```

Comment:
```suggestion
    name: 'Item',
```
Looks like you forgot to change this code to 'Item' for the partition-based-sync state

Reply:
This is actually intentional - the PBS backend still uses `Task` and it's hardcoded into the old backend